src/parachain/loans.ts

```diff
@@ -435,18 +540,29 @@ export class DefaultLoansAPI implements LoansAPI {
         return newMonetaryAmount(amount, rewardCurrency);
     }
 
+    async _getExchangeRate(fromCurrency: CurrencyExt): Promise<ExchangeRate<WrappedCurrency, CurrencyExt>> {
+        const wrappedCurrencyId = this.api.consts.escrowRewards.getWrappedCurrencyId;
+        const wrappedCurrency = tokenSymbolToCurrency(wrappedCurrencyId.asToken);
+        if (isCurrencyEqual(fromCurrency, wrappedCurrency)) {
+            const wrappedCurrencyToBitcoinRate = Big(1); // Use exchange rate of 1 between wrapped BTC and real BTC.
```

Comment:
> // Use exchange rate of 1 between wrapped BTC and real BTC.

As far as I can see, `isCurrencyEqual` checks for ticker equality. In which case would `fromCurrency` or `wrappedCurrency` be actual BTC?

Reply:
Good point, the comment was misleading, will remove it. I meant that exchange rate is expressed against BTC and in case of wrapped BTC, the protocol assumes it has the exactly same value as BTC, right?

Reply:
>  the protocol assumes it has the exactly same value as BTC, right?

It does yeah